app/models/blueprint.rb

```diff
@@ -228,12 +228,12 @@ def update_catalog_items(the_bundle, catalog_items)
   end
 
   def add_entry_points(new_bundle, entry_points, dialog, service_type)
-    entry_points ||= {
-      'Provision'  => ServiceTemplate.default_provisioning_entry_point(service_type),
-      'Retirement' => ServiceTemplate.default_retirement_entry_point
-    }
+    entry_points = {} unless entry_points
+    entry_points['Provision'] ||= ServiceTemplate.default_provisioning_entry_point(service_type)
```

Comment:
Actually, I believe in this case it would be wise to alter the entry_points argument so that the `ui_properties` accurately reflects the changes that are being made to the blueprint